skyreader/utils/handle_map_data.py

```diff
@@ -53,20 +59,35 @@ def extract_map(
             this_map = healpy.ud_grade(this_map, max_nside)
         mask = np.logical_and(~np.isnan(this_map), np.isfinite(this_map))
         equatorial_map[mask] = this_map[mask]
+        results_nside = result.result[f"nside-{nside}"]
 
-        for pixel_data in result.result[f"nside-{nside}"]:
+        for pixel_data in results_nside:
```

Comment:
You should be cautious about introducing a variable that is just a pointer to a dict value (think: mutability and clutter). I see it's used farther down in a special case, but keeping it as `result.result[f"nside-{nside}"]` would suffice.

Going further: Adding a method to the `result` variable's class called `get_nside_result(int)` would offer streamlined access--no string manipulation needed.

Reply:
I created a new method to the result variable's class and I call directly the method